README.md

```diff
@@ -243,12 +246,17 @@ documentation in the
 or in the
 [Java SDK Core document about authentication](https://github.com/IBM/java-sdk-core/blob/master/Authentication.md).
 
-## Code examples
+## Using the SDK
+
+For general IBM Cloud SDK usage information, please see
+[this link](https://github.com/IBM/ibm-cloud-sdk-common/blob/master/README.md).
```

Comment:
Instead of references like "this link" or "here" (above), what do you think of using references that describe what the links point to? Such as `IBM Cloud SDK Common README` or so.

Reply:
It makes sense. I did not bother this only because it was [in the template](https://github.com/IBM/platform-services-java-sdk#using-the-sdk).

Reply:
I totally forgot about the template. I think diverging from it in this case is justified, but let's discuss that in a follow-up PR that I'll open later.